build.gradle

```diff
@@ -364,7 +364,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.65'
+  implementation 'com.github.hmcts:civil-commons:1.0.66-CIV-13134'
```

Comment:
Adding comment so we don't merge this. we need to create release tag and update that